process/sync/storageBootstrap/metaStorageBootstrapper.go

```diff
@@ -118,3 +121,8 @@ func (msb *metaStorageBootstrapper) cleanupNotarizedStorage(metaBlockHash []byte
 		}
 	}
 }
+
+func (msb *metaStorageBootstrapper) applySelfNotarizedHeaders(selfNotarizedHeadersHashes [][]byte) ([]data.HeaderHandler, error) {
+	selfNotarizedHeaders := make([]data.HeaderHandler, 0, len(selfNotarizedHeadersHashes))
```

Comment:
is it ok like this? It will return an empty byte slice of slices

Reply:
Done